### PR TITLE
Closes #120:  Add an .about.yml file

### LINKED ISSUE #120: Generate an about.yml file
https://github.com/18F/about_yml

### PR DESCRIPTION
**Pull request by ccostino**
This changeset provides an .about.yml file for the Dolores Landingham bot so that it can be utilized by the Team API.

### CODE REVIEW

**Comment by ccostino** [comment]
@jessieay, please let me know if there's anything missing or incorrect!

File: .about.yml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+#-
+
+# Technologies used to build the project
+stack:
+- Ruby
+- Rails
+- Clockwork
+- Timecop
+- RSpec
+- Slack API
```
Comment (jessieay) [inline .about.yml]: might add MyUSA Authentication

File: .about.yml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# Brief description of the project's outcomes
+impact: A successful drip campaign of information now supports new hires at 18F.
+
+# Services used to supply project status information
+# Items:
+# - name: Name of the service
+#   category: Type of the service
+#   url: URL for detailed information
+#   badge: URL for the status badge
+#services:
```
Comment (jessieay) [inline .about.yml]: add Travis and Waffle board?

File: .about.yml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# Items by property name pattern:
+#   .*:
+#     name: Name of the license from the Software Package Data Exchange (SPDX): https://spdx.org/licenses/
+#     url: URL for the text of the license
+licenses:
+  dolores-landingham-bot:
+    name: CC0
+    url: https://github.com/18F/dolores-landingham-bot/blob/master/LICENSE.md
+
+# Blogs or websites associated with project development
```
Comment (jessieay) [inline .about.yml]: link to Melody's onboarding blog post?

File: .about.yml (hunk 1 above)
Comment (ccostino) [inline .about.yml]: Will do!

File: .about.yml (hunk 2 above)
Comment (ccostino) [inline .about.yml]: D'oh, yes!

File: .about.yml (hunk 3 above)
Comment (ccostino) [inline .about.yml]: Yes to that too, I knew I was missing something there!  Yours as well, but once it's up. :-)

File: .about.yml (hunk 1 above)
Comment (ccostino) [inline .about.yml]: Hmm, not sure why the update isn't showing up here, but MyUSA Authentication is a part of this list now (viewing the file itself shows it).

File: .about.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+licenses:
+  dolores-landingham-bot:
+    name: CC0
+    url: https://github.com/18F/dolores-landingham-bot/blob/master/LICENSE.md
+
+# Tag to use when blogging about this project
+blog_tag:
+- onboarding
+- how we work
+- culture
```
Comment (ccostino) [inline .about.yml]: Heads up with this:  the .about.yml schema is changing slightly, hence the reason why this is just a list of tags and the actual link is in the links section below.  Please see https://github.com/18F/about_yml/issues/46 for more details.

File: .about.yml [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+- Ruby
+- Rails
+- Clockwork
+- Timecop
+- RSpec
+- Slack API
+- MyUSA Authentication
+
+# Brief description of the project's outcomes
+impact: A successful drip campaign of information now supports new hires at 18F.
```
Comment (gemfarmer) [inline .about.yml]: this description is confusing, imo.
maybe `impact: A successful drip of campaign of information to support new hires at 18f.`?

File: .about.yml (hunk 1 above)
Comment (gemfarmer) [inline .about.yml]: maybe add Cloud.gov

File: .about.yml [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- github: jessieay
```
Comment (gemfarmer) [inline .about.yml]: `id: jessie`

File: .about.yml [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+  role: lead
+- github: gemfarmer
```
Comment (gemfarmer) [inline .about.yml]: `id: brian`

File: .about.yml [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+- github: ccostino
```
Comment (gemfarmer) [inline .about.yml]: `id: carlo.costino`

File: .about.yml [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+- github: monfresh
```
Comment (gemfarmer) [inline .about.yml]: `id: moncef`

File: .about.yml [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+- github: melodykramer
```
Comment (gemfarmer) [inline .about.yml]: `id: melody`

File: .about.yml [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+- github: brittag
```
Comment (gemfarmer) [inline .about.yml]: `id: britta.gustafson`

File: .about.yml [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+- github: danielmorrison
+- github: adelevie
```
Comment (gemfarmer) [inline .about.yml]: `id: alan`

**Comment by gemfarmer** [comment]
@ccostino looks good. made a few line notes, mostly pulling team ids from [here](https://github.com/18F/hub/tree/master/_data/team)

**Comment by ccostino** [comment]
Awesome, thanks @gemfarmer!  I agree about the description being confusing after re-reading it myself.

As for the team IDs, thanks for catching folks I missed!  I think they are supposed to be GitHub usernames since they are associated with the `github` key but I will double check that and swap them for our Team API IDs if need be.

I caught one issue too with the blog post link:  `blog` is not a category - only `api`, `process`, and `related` are valid categories in the schema.  I'll swap it to `process` since the post is about how building the bot was accomplished and its direct impact to the improvement of our onboarding process.

**Comment by ccostino** [comment]
@gemfarmer:  I just updated the description and link category, and I confirmed that it's the GitHub usernames we want in this case given that we are referencing folks via their GitHub account in here.

**Comment by ccostino** [comment]
I rebased this with the latest `master` updates and squashed all of the commits to prepare this for merging. :-)

**Comment by gemfarmer** [comment]
@ccostino I guess I was trying to say that we have values for `github` and `id`.

**Comment by ccostino** [comment]
@brian:  Ah, okay, my mistake for misunderstanding - we could include both to be complete.

**Comment by ccostino** [comment]
@gemfarmer:  IDs have now been added!

**Comment by gemfarmer** [comment]
Looks good! Would you like to do the honors?

**Comment by ccostino** [comment]
Sure thing, thanks @gemfarmer!

**Pull request merged**